### ISSUE TITLE
The fastMRI dataset format is in .h5 but this code loading a data in .mat.

### ISSUE THREAD

**Issue by MNaseerSubhani** [byaman14/ZS-SSL#3]
Can you please explain the mask that we need to train the model, should I need to use validation set from fastMRI?

**Comment by byaman14** [comment]
Hi,

Regarding dataset format: you can change the line of code for .mat with .h5 (we use Scipy for loading .mat, you can replace it with h5py package to load .h5 files).

Regarding validation set: You don't need any additional dataset from fastMRI.  ZS-SSL is a scan-specific approach, requiring a single sample/scan. Training and validation are done on the same sample via 3 way split of acquired measurements (detailed in the paper).

**Comment by MNaseerSubhani** [comment]
Hi, 

I understand the kspace and coil sensitivity map generated from same kspace but did not fully understand the mask with the size of (width, height). Do I need to generate from thresholding of ifft(kspace)?

**Comment by byaman14** [comment]
Hi,
Please see the tutorial for mask generation https://github.com/facebookresearch/fastMRI/blob/main/fastMRI_tutorial.ipynb.

**Comment by MNaseerSubhani** [comment]
Thankyou, I'll look in to it.
